src/qibocal/protocols/characterization/ramsey.py

```diff
@@ -307,24 +280,19 @@ def _plot(data: RamseyData, qubit, fit: RamseyResults = None):
 
     qubit_data = data.data[qubit]
     waits = data.waits
-    msrs = qubit_data[["msr"]].tolist()
-    if data.nboot != 0:
-        msrs = np.reshape(msrs, (len(waits), -1)) * V_TO_UV
-        nsamples = msrs.shape[1]
-        error_bars = np.std(msrs, axis=1).flatten() / np.sqrt(nsamples)
-        msrs = np.mean(msrs, axis=1).flatten()
-    else:
-        msrs = np.reshape(msrs, (len(waits))) * V_TO_UV
-        error_bars = np.zeros(len(msrs))
+    probs = qubit_data[["prob"]].tolist()
```

Comment:
You don't need to make it a list
```suggestion
    probs = qubit_data[["prob"]]
```

You're processing with NumPy even after.

Reply:
Without a `tolist()`,  `probs` will be a recarray and the plot is not rendered as expected.

Reply:
Ok, this is because you're slicing with a list
```suggestion
    probs = qubit_data["prob"]
```

Reply:
Moreover, avoiding the list you probably can skip also the `np.reshape` right after (but about this I'm not sure, it would only if the current shape were `(len(waits), 1)` and you're just reshaping to get rid of the extra dimension).